### PR TITLE
[Site] Clean templates & css

### PR DESCRIPTION
**Pull request by smnandre**
* Clean demo templates
* Remove unused helpers
* Remove unused utilities

### CODE REVIEW

**Comment by smnandre** [comment]
Is it possible to disable fabbot for the website ? cc @kbond @Kocal

**Comment by Kocal** [comment]
I don't know how to, I believe you should ask to Fabien

File: ux.symfony.com/assets/styles/app.scss [hunk 1]
```diff
@@ -11,12 +11,54 @@
...
+
 // Boostrap remainder of required parts
 @import "../../vendor/twbs/bootstrap/scss/maps";
 @import "../../vendor/twbs/bootstrap/scss/mixins";
 @import "../../vendor/twbs/bootstrap/scss/utilities";
 @import "../../vendor/twbs/bootstrap/scss/root";
 
+// Remove multiple utilities with a comma-separated list
+// Margin
+$utilities: map-remove(
```
Comment (javiereguiluz) [inline ux.symfony.com/assets/styles/app.scss:25]: I'm not sure about this kind of micro-optimizations. Reasons:

* The full Bootstrap CSS contents take only 30 KBs when compressed with br or zstd, so it's not a problem for most folks
* If we remove common utilities like `border`, they won't work when we try to use them in the future and debugging why can be frustrating.

Comment (smnandre) [reply ux.symfony.com/assets/styles/app.scss:25]: The bootstrap CSS files is currently 90% useless _because_ of all the utilities we do not use/need  ... and the only failure in the PageSpeed results.

pagespeed

As i explained some time ago, i'll rework the entire HTML of the website, the final goal is to be almost "classless", and thus facilitate content creation/aggregation (mainly for demos but also cookbook, articles, docs, etc..).

I put this work in "pause" after the "dark/light theme" debate .. but i need to move on now, and trimming down the dependencies to the only things we really use is a method i'm use to work with. It's then much easier to move/recode things :) 

So i understand your vision and would agree if this rework project was not in progress :)

**Review by Kocal — APPROVED** [review]

File: ux.symfony.com/assets/styles/app.scss (hunk 1 above)
Comment (javiereguiluz) [reply ux.symfony.com/assets/styles/app.scss:25]: OK then. Thanks for the additional insights! And thanks as always for your work here.

**Review by javiereguiluz — APPROVED** [review]

**Pull request merged**